joplin/base/views/new_page_from_modal.py

```diff
@@ -25,98 +26,100 @@ def import_page_from_url(url, jwt_token):
     return page.id
 
 
+@require_POST
 def new_page_from_modal(request):
     user_perms = UserPagePermissionsProxy(request.user)
     if not user_perms.can_edit_pages():
         raise PermissionDenied
 
-    if request.method == 'POST':
```

Comment:
out of curiosity, what other request.method would be /could have been passed into here?

Reply:
Nothing logical. We aren't making any GET, DELETE, or PUT requests to new_page_from_modal.